### DISCUSSION TITLE
umi4 antd5 全局主题切换

### DISCUSSION THREAD

**Discussion by jintangWang** [ant-design/ant-design#42096] [Q&A] [answered]
umi4 没有入口组件，如果配置全局主题呢？

**Comment by BoyYangzai** [comment, marked as answer]
https://umijs.org/docs/guides/routes#%E5%85%A8%E5%B1%80-layout
@jintangWang cc

**Reply by jintangWang** [reply]
> https://umijs.org/docs/guides/routes#%E5%85%A8%E5%B1%80-layout @jintangWang cc

谢谢你，我最后自定义了个layout组件，用于包裹 page，因为要不同的页面显示不同的 layout，效果还行
